applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -520,29 +526,41 @@ int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate)
 
 int le_audio_volume_up(void)
 {
-	if (active_stream.stream->ep->status.state != BT_AUDIO_EP_STATE_STREAMING) {
-		return -ECANCELED;
+	if (active_stream.stream != NULL) {
```

Comment:
Thanks for the PR!
To reduce indentations, we recommend checking that `if (active_stream.stream == NULL) ` then return an error if so.

Reply:
Suggest to return //EFAULT 14	/* Bad address */ in case the pointer is NULL.

Reply:
Hi @koffes, thank you for your comments! I'll change the code accordingly. Just a quick questions since this is my first PR in this project, what is the preferred way of updating PRs? Should I create a new commit or amend the existing one?

Reply:
Hi @koffes, I've incorporated the required changes. To keep cleaner history I updated the PR using git rebase as suggested by Zephyr contribution guidelines, I hope it's ok. Now if either `active_stream.stream` or `active_stream.stream->ep` are `NULL`, `-EFAULT` is returned.

Reply:
Thanks. Yes, at least for smaller PRs like this, that is a good way to do it.